.github/workflows/build_docker.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish Docker Image
+
+on:

```

Comment:
@qstokkink 
Which triggers are expected for the docker build?

Reply:
Just the `workflow_dispatch`. Eventually, I want this to be part of the `build.yml` to have a stable Docker image per release.